types/feathersjs__socketio/index.d.ts

```diff
@@ -1,12 +1,22 @@
 // Type definitions for @feathersjs/socketio 3.0
 // Project: https://feathersjs.com
 // Definitions by: Jan Lohage <https://github.com/j2L4e>
+//                 Desmond Koh <https://github.com/deskoh>
 // Definitions: https://github.com/feathersjs-ecosystem/feathers-typescript
 // TypeScript Version: 2.8
 
-/// <reference types="feathersjs__socket-commons"/>
-import SocketIO = require("socket.io");
+/// <reference types="node" />
 
-export default function feathersSocketIO(callback?: (io: SocketIO.Server) => void): () => void;
-export default function feathersSocketIO(options: number | SocketIO.ServerOptions, callback?: (io: SocketIO.Server) => void): () => void;
-export default function feathersSocketIO(port: number, options?: SocketIO.ServerOptions, callback?: (io: SocketIO.Server) => void): () => void;
+import * as http from 'http';
```

Comment:
Is there a reason you're importing this? It doesn't look like it's used anywhere.

Reply:
I can't recall the issue it caused previously. But removing it together with `node` reference seemed fine now.